package.json

```diff
@@ -93,13 +96,17 @@
     "ts-node": "^10.9.1",
     "typescript": "^5.2.2",
     "typescript-docs-verifier": "^2.5.0",
-    "webpack": "^5.88.2",
-    "wait-port": "^1.1.0",
-    "vitest": "^0.34.6",
+    "vite-plugin-node-polyfills": "^0.15.0",
+    "vite-plugin-top-level-await": "^1.3.1",
+    "vite-plugin-electron-renderer": "^0.14.5",
+    "vitest": "1.0.0-beta.2",
     "vitest-when": "^0.2.0",
-    "@vitest/coverage-v8": "^0.34.6"
+    "wait-port": "^1.1.0",
+    "webdriverio": "^8.20.0",
+    "webpack": "^5.88.2"
   },
   "resolutions": {
-    "dns-over-http-resolver": "^2.1.1"
+    "dns-over-http-resolver": "^2.1.1",
+    "chai": "^4.3.10"
```

Comment:
As we are still using `chai` as direct dev-dependency of our packages, so have to add a resolution to solve conflict between `vitest` deep dependency.